packages/react/src/components/Input/Input.stories.tsx

```diff
@@ -24,28 +24,28 @@ export default {
     type: {
       control: 'select',
       options: [
+        /* | "button" */
         'checkbox',
+        'color',
         'date',
         'datetime-local',
         'email',
+        'file',
+        'hidden',
+        /* 'image', */
         'month',
         'number',
         'password',
         'radio',
+        /* | "range" */
+        /* | "reset" */
         'search',
+        /* | "submit" */
         'tel',
         'text',
         'time',
         'url',
         'week',
-        // 'button',
-        // 'color',
-        // 'file',
-        // 'hidden',
-        // 'image',
-        // 'range',
-        // 'reset',
-        // 'submit',
       ],
```

Comment:
This should not be needed to define in stories. Storybook should automatically provide a select from deriving the type